app/build.gradle

```diff
@@ -36,7 +36,7 @@ dependencies {
 
   // Get kiwixlib online if it is not populated locally
   if (file("../kiwixlib/src/main").list().length == 1) {
-    compile 'org.kiwix.kiwixlib:kiwixlib:0.0.6'
+    compile 'org.kiwix.kiwixlib:kiwixlib:0.1.0'
```

Comment:
What is this version number ?

It is the official kiwix-lib release or it is yours ?
(kiwix-lib release is 0.2 and we will soon release a new version)

Reply:
Its just an arbitrary number at the moment.

Reply:
I notice this PR is still open, despite the tests passing. I'm curious whether @mgautierfr would prefer the version number to match the native library. If not, can we merge this PR? :)